app/src/__tests__/Users.spec.ts

```diff
@@ -6,118 +6,160 @@ import ConfirmationDialog from '@/components/common/ConfirmationDialog.vue'
 import { library } from '@fortawesome/fontawesome-svg-core'
 import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
 import {
-  faCheckCircle,
-  faInfoCircle,
-  faExclamationTriangle,
-  faCircleXmark,
-  faPlus,
-  faPencil,
-  faTrashCan
+    faCheckCircle,
```

Comment:
something happened with the indents in this file, it looks like there are 4 spaces instead of 2 here and below. Could you please fix it, it will make it easier to review the code so that only new changes are visible.

Reply:
Fixed in recent update.